common/src/types/v0/transport/nexus.rs

```diff
@@ -409,8 +412,18 @@ impl DestroyNexus {
             node,
             uuid,
             disowners: NexusOwners::None,
+            lazy: false,
```

Comment:
Should we take the lazy from parameters here as well like in ShutdownNexus below?

Reply:
Most calls would set it to false anyway, which is why I didn't add it, it can be add with the builder type method.